tests/loader_version_tests.cpp

```diff
@@ -444,6 +442,82 @@ TEST(MultipleDriverConfig, DifferentICDsWithDevicesAndGroups) {
     ASSERT_EQ(group_count, returned_group_count);
 }
 
+#if defined(WIN32)
+TEST(MultipleICDConfig, version_5_and_version_6) {
```

Comment:
Add a comment:  "This is testing the Windows device adapter sorting mechanism with drivers that export EnumerateAdapterXX with drivers that do not expose that".  Or whatever.